### PR TITLE
Implemented new support page.

### PR DESCRIPTION
**Pull request by ulasturann**
### Feature Requests
Implementation of the new PRO Support page.

Fixes https://github.com/primefaces/primereact/issues/3451

### CODE REVIEW

**Review by melloware — APPROVED** [review]

File: pages/support/index.js [hunk 1]
```diff
@@ -11,219 +12,233 @@ const SupportPage = () => {
...
-                                PrimeReact PRO, it's easy to support, tune and add features to PrimeReact as if it were an in-house framework.
-                            </p>
-                        </div>
-                        <a className="action-button" href="mailto:PrimeReact@primetek.com.tr">
-                            GET A QUOTE
-                        </a>
+                    <div className="card m-0 xl:w-9 text-white bg-cover" style={{ backgroundImage: `url(${contextPath}/images/support/card-pro.jpg)` }}>
+                        <div className="text-xl font-semibold mb-3">PRO Support</div>
+                        <p className="m-0 line-height-3">
+                            With PRO support, it's easy to support, tune and add features to PrimeNG as if it were an in- house library. PRO is a term based commercial support service. With the exclusive services of PRO account, it is no longer
```
Comment (melloware) [inline pages/support/index.js:30]: PrimeNG should be PrimeReact

File: pages/support/index.js [hunk 2]
```diff
@@ -11,219 +12,233 @@ const SupportPage = () => {
...
+                                </li>
+                            </ul>
+                        </div>
+                    </div>
+                    <div className="col-12 md:col-6 xl:col-3">
+                        <div className="card m-0 flex-1 h-full">
+                            <div className="text-xl text-900 font-semibold mb-3">Standard</div>
+                            <p className="m-0 line-height-3 mb-3 text-secondary">1 Year Subscription</p>
+                            <a
+                                href="mailto:primeng@primetek.com.tr"
```
Comment (melloware) [inline pages/support/index.js:71]: primeng@primetek.com.tr should be primereact?

File: pages/support/index.js [hunk 3]
```diff
@@ -11,219 +12,233 @@ const SupportPage = () => {
...
+                                </li>
+                            </ul>
+                        </div>
+                    </div>
+                    <div className="col-12 md:col-6 xl:col-3">
+                        <div className="card m-0 flex-1 h-full">
+                            <div className="text-xl text-900 font-semibold mb-3">Premium</div>
+                            <p className="m-0 line-height-3 mb-3 text-secondary">1 Year Subscription</p>
+                            <a
+                                href="mailto:primeng@primetek.com.tr"
```
Comment (melloware) [inline pages/support/index.js:103]: primeng to primereact

**Review by melloware — CHANGES_REQUESTED** [review]

**Review by melloware — APPROVED** [review]

**Comment by melloware** [comment]
@habubey looks like this PR is conflicting now?

**Comment by habubey** [comment]
> @habubey looks like this PR is conflicting now?

I have pushed to updated version commit to the master, please consider it while reviewing.
https://github.com/primefaces/primereact/commit/6fbadf89db29f996b00a8e9daab20f7e9e010577

**Pull request closed without merging**